scripts/setup.sh

```diff
@@ -195,7 +195,7 @@ elif [ "$VAGRANT" = true ]; then
   # XVFB is required for headless testing in Vagrant
   sudo apt-get install xvfb chromium-browser
```

Comment:
Move the xvfb part to run_performance_tests.sh? In general let's put all the xvfb stuff there in order to avoid confusion.